### DISCUSSION TITLE
csv dosyası

### DISCUSSION THREAD

**Discussion by kubrafirat** [gusanmaz/BMB212-Algorithms-2223#25] [Q&A] [answered]
Hocam terminalde kodu çalıştırırken csv dosyası aynı dizinde olmasına rağmen csv dosyası bulunmuyor.Terminalde okutacağımız kodu yazabilir misiniz?

**Comment by gusanmaz** [comment, marked as answer]
Dosyanizin yolunu dogru bir sekilde yazdiysaniz dosyanin bulunmasi lazim. Dosyanizin icerigini okumak icin `readCSV` fonksiyonuna dosyanin yolunu deger olarak vermeniz yeterli.

Gecen senenin ders sayfasindaki asagidaki aciklamalari okumaniz faydali olabilir:
https://github.com/gusanmaz/BMB212_Algorithms#metin-dosyasindaki-degerleri-diziye-aktarma
